src/densities.jl

```diff
@@ -54,15 +57,6 @@ is not collinear the spin density is `nothing`.
     n_k = length(basis.kpoints)
     n_spin = basis.model.n_spin_components
 
-    # Sanity checks
-    @assert n_k == length(ψ)
-    @assert n_k == length(occupation)
-    for ik in 1:n_k
-        @assert length(G_vectors(basis, basis.kpoints[ik])) == size(ψ[ik], 1)
-        @assert length(occupation[ik]) == size(ψ[ik], 2)
-    end
-    @assert n_k > 0
```

Comment:
Not convinced. Quite an elementary piece of the code. Having this helps to spot errors (like wrongful setup of the \psi array).

Reply:
not really though, it's pretty deep inside scf & stuff, so by that point you're expected to have setup this stuff correctly. If these aren't consistent, you'll run into a lot of pretty clear trouble. It's leftover for when we were figuring out this stuff, but not really needed anymore. Otherwise why would we have these checks here and not everywhere else?

Reply:
ok